modules/programs/wlogout.nix

```diff
@@ -0,0 +1,160 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib)
+    all filterAttrs hasAttr isStorePath literalExpression optionalAttrs types;
+  inherit (lib.options) mkEnableOption mkOption;
+  inherit (lib.modules) mkIf mkMerge;
+  inherit (builtins) toJSON concatStringsSep;
+
+  cfg = config.programs.wlogout;
+
+  jsonFormat = pkgs.formats.json { };
+
+  mkMargin = name:
+    mkOption {
+      type = types.nullOr types.int;
+      default = null;
+      example = 10;
+      description = "Margin value without unit.";
+    };
+
+  wlogoutLayoutConfig = with types;
+    submodule {
+      freeformType = jsonFormat.type;
+
+      options = {
+        label = mkOption {
+          type = str;
+          default = "";
+          example = "shutdown";
+          description = "CSS label of button.";
+        };
+
+        action = mkOption {
+          type = str;
+          default = "";
+          example = "systemctl poweroff";
+          description = "Command to execute when clicked.";
+        };
+
+        text = mkOption {
+          type = str;
+          default = "";
+          example = "Shutdown";
+          description = "Text displayed on button.";
+        };
+
+        keybind = mkOption {
+          type = str;
+          default = "";
+          example = "s";
+          description = "Keyboard character to trigger this action.";
+        };
+
+        height = mkOption {
+          type = nullOr (numbers.between 0 1);
+          default = null;
+          example = 0.5;
+          description = "Relative height of tile.";
+        };
+
+        width = mkOption {
+          type = nullOr (numbers.between 0 1);
+          default = null;
+          example = 0.5;
+          description = "Relative width of tile.";
+        };
+
+        circular = mkOption {
+          type = nullOr bool;
+          default = null;
+          example = true;
+          description = "Make button circular.";
+        };
+      };
+    };
+in {
+  meta.maintainers = [ lib.maintainers.Scrumplex ];
```

Comment:
I'm not sure if this has changed, but I believe HM maintainers are still maintained in a separate list in `lib.hm.maintainers`.

If you are already a Nix maintainer, you don't need to add yourself to the list of HM maintainers.